### PR TITLE
Improve callback docs

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Improve docs formatting.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
various callbacks

### CODE REVIEW

File: plugins/callback/counter_enabled.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     extends_documentation_fragment:
       - default_callback
     requirements:
-      - set as stdout callback in ansible.cfg  (stdout_callback = counter_enabled)
+      - set as stdout callback in ansible.cfg (C(stdout_callback = counter_enabled))
```
Comment (russoz) [inline plugins/callback/counter_enabled.py:24]: Maybe 
```suggestion
      - set as stdout callback in C(ansible.cfg) (C(stdout_callback = counter_enabled))
```
On the grounds of being a filename?

File: plugins/callback/logentries.py [hunk 2]
```diff
@@ -13,39 +13,39 @@
...
+        description: Port to use when connecting to the API when TLS is enabled.
         env:
             - name: LOGENTRIES_TLS_PORT
         default: 443
         ini:
           - section: callback_logentries
             key: tls_port
       token:
-        description: The logentries "TCP token"
+        description: The logentries "TCP token".
```
Comment (russoz) [inline plugins/callback/logentries.py:48]:
```suggestion
        description: The logentries C(TCP token).
```

File: plugins/callback/logentries.py [hunk 3]
```diff
@@ -63,7 +63,7 @@
           - section: callback_logentries
             key: use_tls
       flatten:
-        description: flatten complex data structures into a single dictionary with complex keys
+        description: flatten complex data structures into a single dictionary with complex keys.
```
Comment (russoz) [inline plugins/callback/logentries.py:66]:
```suggestion
        description: Flatten complex data structures into a single dictionary with complex keys.
```

File: plugins/callback/slack.py [hunk 4]
```diff
@@ -45,7 +45,7 @@
           - section: callback_slack
             key: username
       validate_certs:
-        description: validate the SSL certificate of the Slack server. (For HTTPS URLs)
+        description: validate the SSL certificate of the Slack server. (For HTTPS URLs.)
```
Comment (russoz) [inline plugins/callback/slack.py:48]:
```suggestion
        description: Validate the SSL certificate of the Slack server. (For HTTPS URLs.)
```

File: plugins/callback/splunk.py [hunk 5]
```diff
@@ -13,22 +13,22 @@
...
     description:
       - This callback plugin will send task results as JSON formatted events to a Splunk HTTP collector.
-      - The companion Splunk Monitoring & Diagnostics App is available here "https://splunkbase.splunk.com/app/4023/"
+      - The companion Splunk Monitoring & Diagnostics App is available here U(https://splunkbase.splunk.com/app/4023/).
       - Credit to "Ryan Currah (@ryancurrah)" for original source upon which this is based.
     requirements:
       - Whitelisting this callback plugin
       - 'Create a HTTP Event Collector in Splunk'
-      - 'Define the url and token in ansible.cfg'
+      - 'Define the URL and token in ansible.cfg'
```
Comment (russoz) [inline plugins/callback/splunk.py:21]:
```suggestion
      - 'Define the URL and token in C(ansible.cfg)'
```

**Review by russoz — COMMENTED** [review]
Some minor adjustments

File: plugins/callback/say.py [hunk 6]
```diff
@@ -14,12 +14,12 @@
     type: notification
     requirements:
       - whitelisting in configuration
-      - the '/usr/bin/say' command line program (standard on macOS) or 'espeak' command line program
+      - the C(/usr/bin/say) command line program (standard on macOS) or C(espeak) command line program
     short_description: notify using software speech synthesizer
     description:
       - This plugin will use the 'say' or 'espeak' program to "speak" about play events.
```
Comment (russoz) [inline plugins/callback/say.py:20]:
```suggestion
      - This plugin will use the C(say) or C(espeak) program to "speak" about play events.
```

File: plugins/callback/slack.py [hunk 7]
```diff
@@ -45,7 +45,7 @@
           - section: callback_slack
             key: username
       validate_certs:
-        description: validate the SSL certificate of the Slack server. (For HTTPS URLs)
+        description: Validate the SSL certificate of the Slack server. (For HTTPS URLs.)
```
Comment (russoz) [inline plugins/callback/slack.py:48]: Not really needed.
```suggestion
        description: Validate the SSL certificate of the Slack server for HTTPS URLs.
```

**Review by russoz — APPROVED** [review]
Couple of minor things and LGTM

**Comment by felixfontein** [comment]
The Alpine CI failure is unrelated.

**Comment by felixfontein** [comment]
@ujenmr @russoz thanks for reviewing this!

**Pull request merged**